hello-tribuo/pom.xml

```diff
@@ -0,0 +1,175 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.gluonhq.samples</groupId>
+        <artifactId>samples</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+
+    <groupId>com.gluonhq.samples</groupId>
+    <artifactId>hellotribuo</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+
+    <name>Hello Tribuo</name>
+
+    <properties>
```

Comment:
These properties (except `main.class`) are already defined by the parent [pom](https://github.com/gluonhq/gluon-samples/blob/master/pom.xml#L12), so they can be removed from here (rename `javafx.plugin.version` to `javafx.maven.plugin.version`)